### ISSUE TITLE
IntelliSense does not automatically open, after upgrade to VS2019 16.6

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#45235]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/content/problem/1069252/intellisense-does-not-automatically-open-after-upg.html)._

---
[regression] [worked-in:16.4]
Steps:
Options -> Text Editor -> C# -> IntelliSense -> Show completion list after a character is typed" is enabled
Open a code file in a new C# project. On a new line, write "Console."

Expected behavior:
IntelliSense appears.

Actual behavior:
IntelliSense does not appear. By manually hitting Ctrl+Space, it does appear.

---
#### Visual Studio Feedback System on 6/9/2020, 01:03 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

#### Vatsalya Agrawal [MSFT] on 6/10/2020, 09:37 AM: 

<p>This issue has been identified as a duplicate of an earlier reported feedback linked here: <a target='_blank' href="https://developercommunity.visualstudio.com/content/problem/1057729/show-items-from-unimported-namespaces-breaks-membe.html">https://developercommunity.visualstudio.com/content/problem/1057729/show-items-from-unimported-namespaces-breaks-membe.html</a>. If you reported or voted here, your vote has been applied to the original issue. Everyone else can add their vote directly at the above linked feedback. Voting helps increase the priority of the issue by consolidating customer impact under one feedback. Thank you!</p>

---
### Original Solutions
(no solutions)

**Comment by CyrusNajmabadi** [comment]
This doesn't repro for me.  My suggestion woudl be to close this and ask the filer for a fuller repro that can allow us to proceed.

**Comment by Pikeman** [comment]
On my initial feedback (which was mistakenly closed as a duplicate of #45042), I attached a report (including screen recording of Visual Studio, its log files etc). Here's the link to that feedback anyhow: https://developercommunity.visualstudio.com/content/problem/1069252/intellisense-does-not-automatically-open-after-upg.html

Is there enough information on that ticket? Otherwise I'll be happy to provide more details.

Thanks

**Comment by CyrusNajmabadi** [comment]
Hrmm.. i don't see a screen recording.  Can you supply one here.  @genlu can you think of anything that woudl prevent this?

**Comment by CyrusNajmabadi** [comment]
Also, does this repro on a brand new project?

**Comment by Pikeman** [comment]
> Hrmm.. i don't see a screen recording. Can you supply one here. @genlu can you think of anything that woudl prevent this?

Hm okay, then I will try it again later! I'm back on that machine in 14 hours or so.

> Also, does this repro on a brand new project?

Yes, I made a number of new .NET Core console applications and had reliable repro.

**Comment by genlu** [comment]
For typing regular chars, [this method ](http://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/Completion/CompletionServiceWithProviders.cs,556) would ultimately decide whether completion should be triggered. But it's unclear to me why it doesn't work when the option "show completion after typed char" is enabled.

**Comment by Pikeman** [comment]
Here's a new screen recording. I made it with YouTube this time, instead of the built-in feedback tool (but I can give that another go as well, if it would be more useful - I think it tries to attach some VS internal log files?). https://youtu.be/Hxg7MMY6J8I

Note: Since yesterday I installed the ReSharper Ultimate extension, but I disabled it for this recording, and IntelliSense behaves exactly as it did: completion list does not appear automatically (and then it disappears on next input). When the completion list appears in the video, it is because I manually hit Ctrl+Space.

Cheers
